persistence-modules/java-mongodb/src/main/java/com/baeldung/MongoBsonExample.java

```diff
@@ -1,14 +1,15 @@
 package com.baeldung;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.bson.Document;
+
 import com.mongodb.client.MongoClient;
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
```

Comment:
let's revert this file, since it does not contain any changes

Reply:
I did the changes